hbt/calibration/default.py

```diff
@@ -7,9 +7,9 @@
 from __future__ import annotations
 
 from columnflow.calibration import Calibrator, calibrator
-from columnflow.calibration.cms.met import met_phi_run2, met_phi
+#from columnflow.calibration.cms.met import met_phi_run2, met_phi
```

Comment:
```suggestion
from columnflow.calibration.cms.met import met_phi_run2, met_phi
```

Still want that for all other years